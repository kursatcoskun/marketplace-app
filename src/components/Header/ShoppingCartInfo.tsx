import { ShoppingCartOutlined } from "@ant-design/icons";
import { CartInfoText } from "./CartInfoText";
import { CenteredDiv } from "./CenteredDiv";
export interface ShoppingCartInfoProps {
  cartTotal: string | number;
}

const ShoppingCartInfo: React.FunctionComponent<ShoppingCartInfoProps> = (
  props
) => {
  return (
    <CenteredDiv>
      <ShoppingCartOutlined style={{ fontSize: "200%" }} />
      <CartInfoText>&#8378;{props.cartTotal}</CartInfoText>
    </CenteredDiv>
  );
};

export default ShoppingCartInfo;
