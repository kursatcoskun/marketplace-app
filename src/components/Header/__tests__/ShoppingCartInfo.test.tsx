// MyComponent.test.js
import { shallow } from "enzyme";
import ShoppingCartInfo from "../ShoppingCartInfo";
describe("ShoppingCartInfoComponent", () => {
  it("should render ShoppingCartInfo component", () => {
    const wrapper = shallow(<ShoppingCartInfo cartTotal={1111} />);
  });

  it("should render initial layout", () => {
    const component = shallow(<ShoppingCartInfo cartTotal={1111} />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
