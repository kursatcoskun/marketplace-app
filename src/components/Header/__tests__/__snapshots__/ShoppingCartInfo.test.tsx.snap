// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShoppingCartInfoComponent should render initial layout 1`] = `
Array [
  <styled.div>
    <ForwardRef(ShoppingCartOutlined)
      style={
        Object {
          "fontSize": "200%",
        }
      }
    />
    <Styled(Text)>
      â‚º
      1111
    </Styled(Text)>
  </styled.div>,
]
`;
